.github/workflows/buildimages-update.yml

```diff
@@ -0,0 +1,97 @@
+name: Update buildimages
+
+on:
+  workflow_dispatch:
+    inputs:
+      images_id:
+        description: 'Images ID'
+        required: true
+        type: string
+      go_version:
+        description: 'Go version'
+        required: true
```

Comment:
if this was not required we could have use it even when we don't modify the go version, wdyt (ie in any buildimage promotion to datadog-agent)

Reply:
Using it for every commit is the end goal honestly !

I considered that even if you don't change the Go version you can still pass it as parameter, that way the use of the workflow is the same no matter if you changed the Go workflow, and only check in the workflow whether the version is the same
- find go version
- find buildimages ID
- trigger workflow with those

But to be fair I think you're right and we can just use the version from the repo (ie. not changed) if it's not provided, I'll edit the workflow

Reply:
I think I'll do that in a follow-up PR so that I can have this one merged today